packages/grid/_modules_/grid/components/panel/Panel.tsx

```diff
@@ -2,46 +2,18 @@ import * as React from 'react';
 import ClickAwayListener from '@material-ui/core/ClickAwayListener';
 import Paper from '@material-ui/core/Paper';
 import Popper from '@material-ui/core/Popper';
-import { makeStyles, Theme } from '@material-ui/core/styles';
 import { ApiContext } from '../api-context';
 import { isMuiV5 } from '../../utils';
-
-const useStyles = makeStyles(
-  (theme: Theme) => ({
-    root: {
-      backgroundColor: theme.palette.background.paper,
-      minWidth: 300,
-      maxHeight: 450,
-      display: 'flex',
-      flexDirection: 'column',
-      flex: 1,
-      '& .MuiDataGridPanel-header': {
-        padding: 8,
-      },
-      '& .MuiDataGridPanel-container': {
-        display: 'flex',
-        flexDirection: 'column',
-        overflow: 'auto',
-        flex: '1 1',
-      },
-      '& .MuiDataGridPanel-footer': {
-        padding: 4,
-        display: 'flex',
-        justifyContent: 'space-between',
-      },
-    },
-  }),
-  { name: 'MuiDataGridPanel' },
-);
+import { useStyles } from '../containers/GridRootStyles';
```

Comment:
I think that this is worse, we inject twice unelated styles. What's wrong with the existing solution?

Reply:
it's discussable.
As the popper is rendered outside the grid container, and we prefix all class with our root class, when we render outside the grid then we cannot use the classes prefixed with the root class. Unless we import it as it is or we manually add the root class to the container, but we will potentially miss some standard grid styles hence why I took this path

Reply:
What's wrong with isolating the styles, keeping them as close as possible to where they are needed? Why do we move them to the GridContainer?

Reply:
if you don't use the Panel then you will not load those styles. So you won't be able to use the container panel or footer outside the Panel component

Reply:
> So you won't be able to use the container panel or footer outside the Panel component

Yes, it's one of the reasons why I think we shouldn't do it.

Reply:
https://deploy-preview-890--material-ui-x.netlify.app/storybook/?path=/story/x-grid-demos-custom-components--custom-panel

This story uses the panel container, footer, and header without the wrapper, so if you don't use Panel you won't get the style. Hence why I put it in root. We can move it and make the user import it, but I'm not sure it will have any benefits as it will already be part of the bundle...